master/setup.py

```diff
@@ -296,7 +296,7 @@ def define_plugin_entries(groups):
             ('buildbot.steps.source.cvs', ['CVS']),
             ('buildbot.steps.source.darcs', ['Darcs']),
             ('buildbot.steps.source.gerrit', ['Gerrit']),
-            ('buildbot.steps.source.git', ['Git', 'GitCommit', 'GitTag']),
+            ('buildbot.steps.source.git', ['Git', 'GitCommit', 'GitTag', 'GitPush']),
```

Comment:
Please add it before `GitTag` to keep them in alphabetic order.

Reply:
Done.